app/controllers/sniffs_controller.rb

```diff
@@ -7,14 +7,16 @@ class SniffsController < ApplicationController
   def create
     @dog = Dog.find(params[:dog_id])
     @sniff = Sniff.new
-    @sniff.sniffer = current_user.dog
+    @sniff.sniffer = current_user.dogs.first
     @sniff.sniffed = @dog
 
     if @sniff.save
       @chatroom = Chatroom.new(sniff: @sniff)
       @chatroom.save
```

Comment:
A sniff should only be created when a dog sniffs back (sniff.status = true).  So we dont need this here, but in den sniff back feature.

Reply:
I will move the chatroom creation to sniff back functionality